app/javascript/components/CourseItem.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  FaGraduationCap,
+  FaClock,
+  FaMoneyBillAlt,
+} from 'react-icons/fa';
+
+const CourseItem = ({ course }) => (
+  <div key={course.id} className="single-course">
+    <img src={course.img_url} alt={course.title} className="d-block w-50 h-50" />
```

Comment:
We should make all images the same size so that UI will be cleaner and more aesthetic. You can do that by setting a fixed width and height. For example: 
```jsx
<img src={url} className="d-block w-100" style={{height: '100 px'}} />
```

Reply:
@IndieCoderMM Thanks for the suggestions